### PR TITLE
i18n(zh-cn): Add `flotiq.mdx`

### PR DESCRIPTION
**Pull request by Nin3lee**
#### Description (required)

Add `flotiq.mdx`

#### Related issues & labels (optional)

- #9798 
- Suggested label: `i18n`

### CODE REVIEW

**Comment by liruifengv** [comment]
> Just commenting for maintainers that my preference would be to change this to merge into `next` and see if it "just works" and if so, let's add it there.
> 
> If not, then I think this one new page should be fine to merge into v4 docs as it doesn't seem to contain any outdated links, or affect any existing pages.

I think we can wait V5 release. Then merge this PR.

**Review by sarah11918 — DISMISSED** [review]
OK! Just putting a block on this so it does not get accidentally merged into v4 docs.

See you in v5, @Nin3lee ! 😄

**Comment by sarah11918** [comment]
OK @Nin3lee ! V5 is out, so I'm taking off the block, but please do update your branch because the site has changed A LOT since this PR was made. 😅  I'm sorry if there might be conflicts to deal with!

File: src/content/docs/zh-cn/guides/cms/flotiq.mdx
Comment (Nin3lee) [inline src/content/docs/zh-cn/guides/cms/flotiq.mdx:1]: cc @liruifengv

**Review by liruifengv — APPROVED** [review]
LGTM! Thank you!

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**